libs/designsystem/src/lib/components/text-link/text-link.component.scss

```
@@ -0,0 +1,36 @@
+@import '../../scss/utils';
+@import '../icon/icon.component.shared';
+
+:host {
+  cursor: pointer;
+
+  a {
+    font-size: font-size('n');
+    line-height: line-height('n');
+    color: get-text-color('black');
+  }
+
+  kirby-icon {
+    padding-left: size('xxxs');
+    padding-right: size('s');
+    vertical-align: middle;
+    font-size: #{icon-font-size('sm')};
+  }
+  &.sm {
+    a {
+      font-size: font-size('s');
+      line-height: line-height('s');
+    }
+  }
+  &.xs {
+    a {
+      font-size: font-size('xs');
+      line-height: line-height('xs');
+    }
+    kirby-icon {
+      padding-left: size('xxxxs');
+      padding-right: size('s');
+      font-size: #{icon-font-size('xs')};
```

Consider using the exposed API instead (kirby-icon's `size` input property or custom css properties):
```suggestion
      --kirby-icon-font-size: #{icon-font-size('xs')};
```